src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/nodes/MappingNode.kt

```diff
@@ -56,7 +57,7 @@ class MappingNode @JvmOverloads constructor(
             val valueNode = when (node.valueNode) {
                 // avoid overflow in case of recursive structures
                 is CollectionNode<*> -> "CollectionNode(size:${node.valueNode.value?.size})"
-                else                 -> node.toString()
+                else                                                                                                                -> node.toString()
```

Comment:
No need to change here I guess.

Reply:
Oops, yes.

IntelliJ tried to be helpful and added FQN to some relocated classes - annoying!

```suggestion
                else                 -> node.toString()
```